scala-package/infer/src/main/scala/org/apache/mxnet/infer/ObjectDetector.scala

```diff
@@ -94,13 +94,13 @@ class ObjectDetector(modelPathPrefix: String,
   def objectDetectWithNDArray(input: IndexedSeq[NDArray], topK: Option[Int])
   : IndexedSeq[IndexedSeq[(String, Array[Float])]] = {
 
-    val predictResult = predictor.predictWithNDArray(input)(0)
+    val predictResult = predictor.predictWithNDArray(input)(0).asInContext(Context.cpu())
```

Comment:
If possible, can we apply this to ```ImageClassifier``` class as well ? 
Maybe this can provide a speed boost there as well ?

Reply:
Same nitpick I had on imageclassifier. A comment explaining this wouldn't hurt since it's not obvious why we're doing this.